lib/gtfs/direction.ex

```diff
@@ -1,7 +1,50 @@
 defmodule Gtfs.Direction do
+  alias Gtfs.{Csv, Route}
+
   @type id :: 0 | 1
 
-  @spec from_string(String.t()) :: id()
-  def from_string("0"), do: 0
-  def from_string("1"), do: 1
+  @type t :: %__MODULE__{
+          route_id: Route.id(),
+          direction_id: id(),
+          direction_name: String.t(),
+          direction_destination: String.t()
```

Comment:
`direction_destination` isn't used anywhere. Unless we think we'll use it sometime soon, we shouldn't bother loading it.